package.json

```diff
@@ -11,8 +11,10 @@
     "amman:stop": "./node_modules/.bin/amman stop"
   },
   "dependencies": {
+    "@coral-xyz/xnft-cli": "^0.6.8-latest.5333",
     "@project-serum/anchor": "^0.26.0",
-    "anchor-lang": "^0.0.1-security"
+    "anchor-lang": "^0.0.1-security",
+    "react-xnft": "^0.2.0-latest.3375"
```

Comment:
I don't think xNFT packages are used in this repo.

Reply:
fixed in next commit